libraries/lib-wave-track/WaveClip.cpp

```diff
@@ -1737,6 +1747,11 @@ double WaveClip::SnapToTrackSample(double t) const noexcept
    return std::round(t * mRate) / mRate;
 }
 
+size_t WaveClip::GetNextId()
+{
+    return sNextId++;
```

Comment:
Maybe we should make 0 - not a valid id (we can't do that with indexes, but we can with ids).
And then you need to change it to `return ++sNextId;`